packages/react-components/src/components/Survey/StarSurvey/StarSurvey.tsx

```diff
@@ -0,0 +1,192 @@
+  /**
+   * Helper text displayed below survey question after user select four star
+   */
+  starSurveyFourStarText?: string;
+  /**
+   * Helper text displayed below survey question after user select five star
+   */
+  starSurveyFiveStarText?: string;
+  /**
+   * Confirm Button Label
+   */
+  starSurveyConfirmButtonLabel?: string;
+  /**
+   * Aria Label for each individual star rating
+   */
+  starRatingAriaLabel?: string;
+  /**
+   * Aria Label for cancel button
+   */
+  cancelButtonAriaLabel?: string;
+}
+
+/**
+ * Props for {@link StarSurvey} component.
+ *
+ * @internal
+ */
+export interface _StarSurveyProps {
+  /**
+   * Custom icon name for selected rating elements.
+   * @defaultvalue FavoriteStarFill
+   */
+  selectedIcon?: string;
+  /**
+   * Custom icon name for unselected rating elements.
+   * @defaultvalue FavoriteStar
+   */
+  unselectedIcon?: string;
+  /** Function to send StarSurvey results*/
+  onSubmitSurvey?: (survey: _CallSurvey) => Promise<_CallSurveyResponse | undefined>;
+  /** Function to close star survey modal*/
+  onDismissStarSurvey?: () => void;
+  /** Star survey strings */
+  strings?: _StarSurveyStrings;
+}
+
+/**
+ * A component to allow users to send numerical ratings regarding call quality
+ *
+ * @internal
+ */
+export const _StarSurvey = (props: _StarSurveyProps): JSX.Element => {
+  const { onSubmitSurvey, onDismissStarSurvey, selectedIcon, unselectedIcon, strings } = props;
+
+  const [rating, setRating] = useState(0);
+
+  const [helperText, setHelperText] = useState(strings?.starSurveyHelperText);
+
+  const theme = useTheme();
+
+  const onRatingChange = useCallback(
+    (ev: React.FormEvent<HTMLElement>, rating?: number | undefined): void => {
+      if (rating) {
+        setRating(rating);
+
+        switch (rating) {
+          case 1:
+            setHelperText(strings?.starSurveyOneStarText);
+            break;
+          case 2:
+            setHelperText(strings?.starSurveyTwoStarText);
+            break;
+          case 3:
+            setHelperText(strings?.starSurveyThreeStarText);
+            break;
+          case 4:
+            setHelperText(strings?.starSurveyFourStarText);
+            break;
+          case 5:
+            setHelperText(strings?.starSurveyFiveStarText);
+            break;
+          default:
+            break;
+        }
+      }
+    },
+    [
+      strings?.starSurveyOneStarText,
+      strings?.starSurveyTwoStarText,
+      strings?.starSurveyThreeStarText,
+      strings?.starSurveyFourStarText,
+      strings?.starSurveyFiveStarText
+    ]
+  );
+
+  const onDismiss = useCallback((): void => {
+    if (onDismissStarSurvey) {
+      onDismissStarSurvey();
+    }
+  }, [onDismissStarSurvey]);
+
+  const onConfirm = useCallback(async (): Promise<void> => {
+    if (onSubmitSurvey) {
+      await onSubmitSurvey({
+        overallRating: { score: rating }
+      }).then(()=> console.log('Survey Result submitted')).catch((e)=> console.log(e));
```

Comment:
remove console log and catch here?

Reply:
Do we want the log here to show survey submitted successfully and show errors if not submitted? This is help contoso debug

Reply:
If there is an issue from the submission wont the SDK throw something for us to handle in the CallAdapter? we shouldn't have to do anything in the component I think